packages/std/src/entry_points.rs

```diff
@@ -0,0 +1,137 @@
+/// This macro generates the boilerplate required to call into the
+/// contract-specific logic from the entry-points to the WASM module.
+///
+/// It macro should be invoked in a global scope, and the argument to the macro
```

Comment:
- "It macro should" is a bit broken language-wise
- What exactly is global scope? Looing at `packages/std/src/exports.rs`, those exports can be created in any module

Reply:
>It macro should

Oops, forgot to remove the "macro" part.

>What exactly is global scope

Here i meant module-global scope. That is, not inside a function or trait, for example.

Reply:
addressed in 0f33c07